From dd1add729a308c3523213d4febc5dca6d8bde39b Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:11:31 +0800
Subject: [PATCH] net/bpfilter/bpfilter_kern.c: remove superfluous headers

bpfilter_kern.c hasn't use any macro or function declared in linux/file.h,
linux/sched.h and linux/umh.h.
Thus, these files can be removed from bpfilter_kern.c safely without
affecting the compilation of the ./net module
---
 net/bpfilter/bpfilter_kern.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/net/bpfilter/bpfilter_kern.c b/net/bpfilter/bpfilter_kern.c
index 51a941b56..15e04e302 100644
--- a/net/bpfilter/bpfilter_kern.c
+++ b/net/bpfilter/bpfilter_kern.c
@@ -2,12 +2,9 @@
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 #include <linux/init.h>
 #include <linux/module.h>
-#include <linux/umh.h>
 #include <linux/bpfilter.h>
-#include <linux/sched.h>
 #include <linux/sched/signal.h>
 #include <linux/fs.h>
-#include <linux/file.h>
 #include "msgfmt.h"
 
 extern char bpfilter_umh_start;
-- 
2.25.1

